keps/sig-api-machinery/4153-declarative-validation/README.md

```diff
@@ -776,6 +779,7 @@ interpreted CEL behind the scenes for certain hot code paths:
   new extensions added to the schema.
   - `x-kubernetes-validations`. This field is published already for CRDs, so 
     most clients should already be aware of handling it.
+      - Must support multi-line expression entry, and multiple lines for reason, code, fieldpath of a single rule
```

Comment:
Note that we maybe don't HAVE TO retain multi-line text from IDL to openapi.  If the syntax is not whitespace-sensitive, we can canonicalize into a one-line format, and maybe offer a `celfmt` library for later rendering.

Reply:
Added